tests/e2e/performance/csi_tests/test_pvc_bulk_clone_performance.py

```diff
@@ -239,8 +240,12 @@ def test_bulk_clone_performance(self, namespace, tmp_path):
 
             # Write the test results into the ES server
             full_results.es_write()
```

Comment:
you need to add here the line
`self.results_path = get_full_test_logs_path(cname=self)`

Reply:
Updated the code